repair_sale_order/models/repair_order.py

```diff
@@ -62,7 +62,12 @@ def action_create_sale_order(self):
             sale_order_data = rec._get_sale_order_data()
             sale_order = order_model.create(sale_order_data)
             orders |= sale_order
+            partner_shipping_id = False
```

Comment:
why not do the same with partner_invoice_id?

Reply:
You're right, I added same behavior, thank you